lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -138,6 +138,7 @@ def add_credentials(post, command)
 
       def add_transaction_elements(post, type, options)
         transaction = {}
+        transaction[:paymentCountry] = options[:country] || options[:billing_address][:country] if (options[:billing_address] && options[:billing_address][:country])
```

Comment:
This is the only place using `options[:country]`. Elsewhere `options[:payment_country]` is used (lines 169, 182, 192, 227, and 228).

Reply:
@deedeelavinder Good catch, went back and forth on this a few times and overlooked the final field name. Thanks!